src/routing/wrappers/Layout.svelte

```diff
@@ -14,6 +14,9 @@
   import ProbeDetails from '../../components/regions/ProbeDetails.svelte';
 
   import { store } from '../../state/store';
+
+  let isHomePageView = true;
```

Comment:
It looks like we don't need this `let`.

Reply:
Yea I suppose this should be `isProbeDetailsView` instead so we don't have to check the inverse.